scripts/flash/index.js

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env node
```

Comment:
This whole file needs to be converted to ES5 syntax.

Reply:
updated file to be ES5 format